contracts/upgradeable_contracts/BridgeValidators.sol

```diff
@@ -4,10 +4,14 @@ import "./Ownable.sol";
 import "../IBridgeValidators.sol";
 import "../libraries/SafeMath.sol";
 import "../upgradeability/EternalStorage.sol";
+import "../libraries/Version.sol";
 
 
 contract BridgeValidators is IBridgeValidators, EternalStorage, Ownable {
     using SafeMath for uint256;
+
+    Version.Version public getBridgeInterfacesVersion = Version.Version(1, 0, 0);
```

Comment:
`getBrdigeValidatorsInterfacesVersion` should be here.

Also, it seems that the code was changed slightly from the version we have in https://github.com/poanetwork/poa-bridge-contracts/releases/tag/1.0. But lets inline the version of validators contract with the version of bridge contracts - `2.0.0` should be here.

Reply:
Updated name to `getBridgeValidatorsInterfacesVersion` and version to `2.0.0`